src/Controller/EnMarche/DonationController.php

```diff
@@ -133,4 +137,44 @@ public function resultAction(Request $request, Donation $donation)
             'retry_url' => $retryUrl,
         ]);
     }
+
+    /**
+     * @Route(
+     *     "/mensuel/cancel",
+     *     defaults={"_enable_campaign_silence"=true},
```

Comment:
why this?

Reply:
to have same options with the others controllers, but I don't know exactly why this option

Reply:
if don't know, don't add, no? or check what is it and decide :)

Reply:
ok ^^,